.github/workflows/test.yaml

```diff
@@ -16,10 +16,12 @@ jobs:
     - uses: ./.github/actions/yarn-install
     - run: yarn build
     - run: yarn lint:nofix
+      env: { RD_LINT_SKIP_SPELLING: 1 }
```

Comment:
Wouldn't it be better to default this to `false`, and only set it to `true` in the one workflow that wants to run it?

Reply:
No, because it's default to run for people running `yarn lint` normally.

Reply:
It would still run for people manually because they should not have `CI` set.

The condition would be

```
if [ -n "$CI" && -z "$RD_LINT_SPELLING" ]; then
  echo "Skipping spell checker in CI because RD_LINT_SPELLING isn't set"
  exit
fi
```

Reply:
Good point, I'll do that.